Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -60,6 +63,9 @@
     <PackageReference Include="System.ValueTuple" Version="4.4.0" />
     <Reference Include="System.Configuration" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net45'">
```

Comment:
Can we use `AsyncLocal` from `System.Threading` instead of introducing a dependency on `Microsoft.VisualStudio.Threading`?

Reply:
Fixed